ci/infra/openstack/generate-cloud-provider-conf

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+umask 0077
+
+cat << EOF > openstack.conf
+[Global]
+auth-url=
+username=
+password=
+tenant-name=
+tenant-id=
+domain-name=
+domain-id=
+region=
+ca-file=
+[LoadBalancer]
+lb-version=v2
+subnet-id=
+floating-network-id=
+create-monitor=yes
+monitor-delay=1m
+monitor-timeout=30s
+monitor-max-retries=3
+[BlockStorage]
+trust-device-path=false
+bs-version=v2
+ignore-volume-az=true
+EOF
+
+umask 0022
+
+sed -i '/^auth-url=/ s|=.*|="'${OS_AUTH_URL}'"|
```

Comment:
why not use variable substitution in the cat command above instead of using sed once the file is created?

Reply:
Thank you, you are right. I have changed that